packages/cli/src/config/auth.test.ts

```
@@ -0,0 +1,438 @@
+
+import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest';
+import { 
+  validateAuthMethod, 
+  AuthMethod, 
+  isValidApiKey, 
+  getAuthConfig,
+  saveAuthConfig,
+  clearAuthConfig,
+  AUTH_METHODS
+} from './auth.js';
+import * as fs from 'fs/promises';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)